flocker/node/test/test_loop.py

```diff
@@ -37,6 +39,18 @@ def build_protocol():
     return p
 
 
+def to_node(node_state):
+    """
+    Convert a ``NodeState`` to a corresponding ``Node``.
+
+    :param NodeState node_state: Object to convert.
+    :return Node: Equivalent node.
+    """
+    return Node(hostname=node_state.hostname,
+                applications=node_state.applications,
+                manifestations=node_state.manifestations)
```

Comment:
This is not a public API, this is a utility to make testing easier. Or rather, make tests that used to rely on `NodeState.to_node` continue to work. "Don't do that" basically :grinning: